### PR TITLE
feat(*): add standardised EventArc events

### PR DESCRIPTION
**Pull request by cabljac**
This PR adds some standardised EventArc events to some extensions, as part of https://github.com/firebase/extensions/issues/1135

### CODE REVIEW

File: storage-resize-images/functions/src/index.ts [hunk 1]
```diff
@@ -183,8 +175,10 @@ const generateResizedImageHandler = async (
 
 export const generateResizedImage = functions.storage
   .object()
-  .onFinalize(async (object) => {
+  .onFinalize(async (object, context) => {
+    // await events.recordStartEvent({ object, context });
     await generateResizedImageHandler(object);
+    await events.recordCompletionEvent({ context });
```
[the remark this answers is not in the document]
Comment (cabljac) [reply storage-resize-images/functions/src/index.ts:181]: context i think, according to the completion event spec: [see here?](https://github.com/firebase/extensions/pull/1424/files/4844a5a4bbb46dacca00d165a72e3708ce60bec1#diff-cf28a2adb596945c2977f32cf00850df05eb5fee813b0cd976d4e9ead5250da3R355)

**Review by dackers86 — APPROVED** [review]

**Comment by maihengxing999** [comment]
feat(*): add standardised EventArc events

**Review by raw1359 — APPROVED** [review]

**Comment by maihengxing999** [comment]
next

**Comment by maihengxing999** [comment]
#1424

**Review by ghost — CHANGES_REQUESTED** [review]
Delete off my system

**Review by zjxdnvpzpn-beep — COMMENTED** [review]
@

**Review by sumikn233-al — APPROVED** [review]

**Pull request merged**